hardhat.config.ts

```diff
@@ -68,7 +68,7 @@ const config: HardhatUserConfig = {
 
       // polygon
       polygon: process.env.POLYSCAN_API_KEY || "",
-      polygonMumbai: process.env.POLYSCAN_API_KEY || "",
+      polygonAmoy: process.env.OKLINK_API_KEY || "",
```

Comment:
note: the contract verification didn't work—i can dig into it later but not required atm